src/useChunk.js

```diff
@@ -14,22 +14,51 @@ export function useChunk(defaultContent, { identifier, type }) {
   let fallbackChunk;
   if (typeof defaultChunks !== 'undefined') {
     fallbackChunk = useMemo(
-      () => defaultChunks.find(chunkItem => chunkItem.identifier === identifier),
+      () => {
+        if (identifier) {
+          return defaultChunks.find(chunkItem => chunkItem.identifier === identifier);
+        } else {
+          return defaultChunks.find(chunkItem => chunkItem.content_key === contentKey && chunkItem.project_id == projectId);
+        }
+      },
       [defaultChunks, identifier]
     );
   }
-  const url = identifier
-    ? `chunks/${identifier}`
-    : `chunks/${contentKey}?project_id=${projectId}`;
 
+  let chachedId = identifier || contentKey + projectId;
```

Comment:
Looks like a small typo? Should be cacheId I think :)

Reply:
ahh yep! Thanks